.github/workflows/test-pep273-compatibility.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: '${{ matrix.os }}'
     strategy:
       matrix:
-        python: [ 3.6, 3.9 ]
+        python: [ 3.7, 3.8, 3.9 ]
```

Comment:
This wasn't working as expected so just made it explicit - GitHub actions supports a range syntax but I might be missing something?